Core/COObject.m

```diff
@@ -868,15 +902,19 @@ - (id)valueForVariableStorageKey: (NSString *)key
 	id value = [_variableStorage objectForKey: key];
 	
 	// Convert value stored in variable storage to a form we can return to the user
-	if (value == [NSNull null])
+	if (value == nil)
+	{
+		return aNotFoundMarker;
+	}
+	else if (value == null)
```

Comment:
This is a bit confusing at first glance, maybe rename the cached NSNull instance from `null` to `static_nsnull` or something?